gen.py

```diff
@@ -146,14 +219,14 @@ class Config:
 
 
 FOREACH_DESC = """\
-Iterable to loop through in foreach. Can be a parametrized string, list \
-or a dictionary.
+Iterable to loop through in foreach. Can be a parametrized string, sequence or \
+a mapping.
```

Comment:
To reiterate/summarize (applies to many places), "list" and "dictionary" seem more established. See https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html for example.